cocoa/neural/models.py

```diff
@@ -91,9 +92,8 @@ def forward(self, src, lengths=None, encoder_state=None):
 
         emb = self.embeddings(src)
         s_len, batch, emb_dim = emb.size()
-        mean = emb.mean(0).expand(self.num_layers, batch, emb_dim)
+        encoder_final = emb.mean(0).expand(self.num_layers, batch, emb_dim)
         memory_bank = emb
```

Comment:
why do you change encoder_final here?

Reply:
Originally, encoder_final was a tuple of (mean, mean) which seemed redundant, but looking back, it probably has some purpose.